src/core/client/utility/switch.ts

```diff
@@ -8,9 +8,9 @@ import { sleep } from './sleep';
  * @param {number} duration
  * @return {*}  {Promise<boolean>}
  */
-export async function switchInPlayer(duration: number): Promise<boolean> {
+export async function switchInPlayer(duration: number, switchType: number = 2): Promise<boolean> {
```

Comment:
switchType could be represented by a const to help identify the type...

ie.
```ts
export enum SWITCH_TYPE {
 SOMETHING = 1
 SOMETHING_ELSE = 2
}
```

Helps with auto-completion.

Reply:
I will experiment and try to come up with explicit enum names